build.ps1

```diff
@@ -87,7 +87,7 @@ function Invoke-Build([string]$Path, [switch]$Clean, [string]$Command, [switch]$
             Install-RustToolchain $toolchain
             rustup component add --toolchain $Toolchain rustfmt
             Setup-Environment
-            Invoke-Expression "cargo +$ToolChain $Command --all -- --check"
```

Comment:
Why the removal of `-- --check` here?

Reply:
I want it to actually change the file and not just report what needs to change.